requirements/edx/base.txt

```diff
@@ -58,7 +58,7 @@ edx-oauth2-provider==1.2.0
 edx-opaque-keys==0.4.0
 edx-organizations==0.4.4
 edx-rest-api-client==1.7.1
-git+https://github.com/Stanford-Online/edx-search.git@e2aba4a30cf2dc6aa901799596161a59fa3e3baa#egg=edx-search==1.2.1
+git+https://github.com/Stanford-Online/edx-search.git@5c2321a10f454c1b50aca8e0f53ff26cff2ffa16#egg=edx-search==1.2.2
```

Comment:
Does this SHA need updated?
Maybe point it at the merge commit once search lands?

Reply:
Also, version is definitely outdated :)